packages/hmr-plugin/src/utils/externals.ts

```diff
@@ -1,7 +1,7 @@
-import { HRM_RUNTIME } from '../symbols';
+import { HmrRuntime } from '../symbols';
 
 declare const window: any;
 
-export function hrmIsReloaded(): boolean {
-  return !!(window[HRM_RUNTIME.STATUS] && window[HRM_RUNTIME.STATUS].hmrReloaded);
+export function isHmrReloaded(): boolean {
```

Comment:
I actually prefer the name `hmrIsReloaded`. I think it is more discoverable (by typing `hmr...`) and it reads better when used in an if statement:
```ts
if (hmrIsReloaded()) {
  // ...
}
```

Reply:
done